pyaedt/application/Design.py

```diff
@@ -2244,30 +2291,26 @@ def set_active_design(self, name):
 
     @aedt_exception_handler
     def update_registry_from_file(self, filename):
-        """Update HPC options from an af from an acf configuration file.
+        """Update HPC options from a configuration file.
 
         Parameters
         ----------
-        filename :
-            full path to the filename (can be an acf or a txt)
-
-        Returns
-        -------
-        type
-            nothing
-
+        filename : str
+            Full path and name of the configuration, which can be an ACF or TXT file.
+        
         """
         self._desktop.SetRegistryFromFile(r'%s' % os.path.abspath(filename))
         return True
 
     @aedt_exception_handler
     def validate_simple(self, logfile=None):
-        """Validate Design. Return 0 if validation failed or 1 if passed
+        """Validate a design. Return 0 if validation failed or 1 if passed
```

Comment:
```suggestion
        """Validate a design.
```

Implementation details should not be in the short description.